devel/ansible/roles/dev/files/.bashrc

```diff
@@ -37,6 +37,11 @@ function btest {
     pushd /home/vagrant/bodhi && tox && py.test $@; popd
 }
 
+function btest3 {
+    find /home/vagrant/bodhi -name "__pycache__" -type d -exec rm -r {} \;
+    pushd /home/vagrant/bodhi && py.test-3 $@ && tox; popd
+}
```

Comment:
I would probably make ```btest``` run the Python 3 tests, and maybe make a ```btest2``` if you want. We can rely on CI to run them both. On the other hand, this is more similar to the Fedora packaging guidelines, so I can see why you did it this way. Either way is fine with me, whatever you prefer.

Reply:
We can change it later and publish this change also in docs/readme etc.